### ISSUE TITLE
Volume errordeleting when delete all pvc on Kubernetes

### ISSUE THREAD

**Issue by imquanquan** [opensds/opensds#577]
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind bug

**What happened**:

I created a StatefulSet on Kubernetes with 5 PVs. then I delete the StatefulSet and used ``kubectl delete pvc --all`` to delete all of PVs,but OpenSDS showed that one of volume errordeleting：

```
+--------------------------------------+------------------------------------------+-------------+---------+------+------------------+---------------+--------------------------------------+--------------------------------------+
| Id                                   | Name                                     | Description | GroupId | Size | AvailabilityZone | Status        | PoolId                               | ProfileId                            |
+--------------------------------------+------------------------------------------+-------------+---------+------+------------------+---------------+--------------------------------------+--------------------------------------+
| c96246a9-e5fc-4eda-a846-f74a6f46989c | pvc-e9b440f0-9ea6-11e8-86d5-525400245213 |             |         | 1    | default          | errorDeleting | 2343cb68-f6d9-568b-8637-2da08c02c6dd | 53e11629-7bf9-471b-b3aa-a21e4b0b80df |
+--------------------------------------+------------------------------------------+-------------+---------+------+------------------+---------------+--------------------------------------+--------------------------------------+
```

Then  I do some tests of this.Find out when I delete volumes one by one,all volume will detele correctly:

```
for i in `seq 0 n`; do kubectl delete pv xxx ; sleep 1; done
```

**What you expected to happen**:

All volume detele correctly.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Hotpot(release/branch) version: v0.2.0
- OS (e.g. from /etc/os-release):Debian 9
- Kernel (e.g. `uname -a`): 3.16.0-6-amd64

**Comment by leonwanghui** [comment]
Any update on this issue?

**Comment by imquanquan** [comment]
The problem still exists, many rbd can't be deleted normally

**Comment by imquanquan** [comment]
I closed this issues,and test it on hotpot v0.5.1 later.I will reopen it if there are still some problems.
